src/shared/utils/__tests__/utils.test.ts

```diff
@@ -363,4 +363,50 @@ describe("Util Functions", () => {
       expect(utils.isDocumentRevokable(v3WrappedDidDocument)).toStrictEqual(false);
     });
   });
+
+  describe("getDocumentData", () => {
+    it("should return document data for v2 wrapped document", () => {
+      const v2WrappedDocument = v2WrappedVerifiableDocument as WrappedDocument<OpenAttestationDocument>;
+      expect(utils.getDocumentData(v2WrappedDocument)).toMatchInlineSnapshot(`
+        Object {
+          "$template": Object {
+            "name": "main",
+            "type": "EMBEDDED_RENDERER",
+            "url": "https://tutorial-renderer.openattestation.com",
+          },
+          "issuers": Array [
+            Object {
+              "documentStore": "0x8bA63EAB43342AAc3AdBB4B827b68Cf4aAE5Caca",
+              "identityProof": Object {
+                "location": "demo-tradetrust.openattestation.com",
+                "type": "DNS-TXT",
+              },
+              "name": "Demo Issuer",
+            },
+          ],
+          "recipient": Object {
+            "name": "John Doe",
+          },
+        }
+      `);
+    });
+
+    it("should return document data for v3 wrapped document", () => {
+      const v3WrappedDocument = v3WrappedVerifiableDocument as WrappedDocument<OpenAttestationDocument>;
+      expect(utils.getDocumentData(v3WrappedDocument)).not.toHaveProperty("proof");
```

Comment:
Your don't test the rest is here

Reply:
added match inline. the object contains a default property that kept its original state before omit. I would like to ask if it is okay for the property to remain or we should remove that property before returning it back?

Reply:
nah it makes the test noisy. I just noticed how the json files are imported. Does `import bla from "x.json"` fix the issue? (no `import * as bla`)

Reply:
okay, i update the create separate omit function that removes proof and default. changing to import bla from "x.json" doesn't fix the issue, the data is generalised as an object after importing, but the function only accepts WrappedDocument<OpenAttestationDocument>. i haven't found anyway to typecast during import